docs/scripts/utils.py

```diff
@@ -2,6 +2,7 @@
 
 import pathlib
 import shutil
+import textwrap
```

Comment:
is this a new external dependency? if so it needs to be part of `docs`

Reply:
No, this is part of the standard library: https://docs.python.org/3/library/textwrap.html

Reply:
ok